pandas/tests/indexes/test_base.py

```diff
@@ -1405,6 +1421,25 @@ def test_conversion_preserves_name(self):
         self.assertEqual(i.name, pd.to_datetime(i).name)
         self.assertEqual(i.name, pd.to_timedelta(i).name)
 
+    def test_repeat(self):
+        rep = 2
```

Comment:
these should be in ``Base`` and use ``.create_index()`` so all index types are created. We don't want to add things to ``TestIndex`` if possible (and instead prefere more generic testing)